packages/bpk-component-datatable/src/BpkDataTableHeader.module.scss

```
@@ -16,24 +16,25 @@
  * limitations under the License.
  */
 
-@import '~bpk-mixins/index';
+@import '~bpk-mixins';
 
 $bpk-spacing-v2: true;
 
 $bpk-margin-size-sort-icons: bpk-spacing-sm() + bpk-spacing-md();
 
-.bpk-data-table-column {
+.bpk-data-table-column__header {
+  display: flex;
+  height: 100%;
   margin: bpk-spacing-base();
-  text-overflow: ellipsis;
-  white-space: nowrap;
-  overflow: hidden;
-}
+  flex-direction: row;
+  align-items: center;
+  outline-offset: -(4 * $bpk-one-pixel-rem);
```

Can we just use `bpk-spacing-sm()` as this is 4px?

```suggestion
  outline-offset: -(bpk-spacing-sm());
```